docs/historical_specification.rst

```diff
@@ -0,0 +1,668 @@
+type operator that can be used in place of the ampersand operator:
+
+::
+
+    # generating intersections using the ampersand operator in new versions of Python
+    def f(value: A & B): ...
+
+    # generating intersections using `Intersection` in older versions of Python
+    def f(value: Intersection[A, B]): ...
+
+
+Order and Emptiness
+-------------------
+As for unions the order of elements of an intersection does not matter.
+
+
+`isinstance` and `issubclass`
+-----------------------------
+
+Similarly to union types (see PEP-604), the new syntax should be valid to use in ``isinstance`` and
+``issubclass`` calls, as long as the intersected types are valid arguments to ``isinstance`` and
+``issubclass``.
+
+The `isinstance` or `issubclass` check for an intersection is equal to the combined checks of all
+arguments passed:
+
+::
+
+    class A: ...
+    class B: ...
+
+    assert isinstance(val, A & B) == isinstance(val, A) and isinstance(val, B)
+    assert issubclass(val, A & B) == issubclass(val, A) and issubclass(val, B)
+
+
+It shall be noted, that following the `PEP 544 <https://peps.python.org/pep-0544/#support-isinstance-checks-by-default>`_ about the rejected default ``isinstance`` check:
+If any Protocol within the intersection isn't marked with ``typing.runtime_checkable``,
+``isinstance`` will raise a TypeError.
+
+
+So one possibility to fulfill an intersection is for a class to be a child of all intersected classes.
+
+::
+    class C(A, B): ...
+
+    isinstance(C(), A & B)  # True
+    issubclass(C, A & B)  # True
+
+Basic Reductions
+----------------
+In order for the following rules intended for type checkers to work correctly the following
+reduction have to be applied to Intersections first:
+
+- Nested intersections shall be flattened, i.e ``Intersection[A, Intersection[B, C]] ==
+  Intersection[A, B, C]``
+- If a (concrete or protocol) type ``A`` is a subtype of ``B``, ``A`` shall be removed from the
+  intersection
+- If a protocol ``BP`` defines **all** methods and properties of a protocol ``AP``, ``AP`` shall be
+  removed from the intersection
+- If the concrete class ``A`` fulfils the Protocol ``AP``, ``AP`` shall be removed from the
+  intersection
+- An intersection with only one element shall be normalized to the element.
+
+
+``Any`` Reduction
+-----------------
+As `PEP 483`_ already suggested: ``Any`` shall be removed from an ``Intersection``, i.e.
+``Intersection[A, B, Any] == Intersection[A, B]``.
+
+% This is only a suggestion and needs to be discussed and decided in https://github.com/CarliJoy/intersection_examples/issues/1
+% Once it was finally decided the discussion and arguments should be summarized here.
+
+
+``Never`` Evaluation
+--------------------
+An intersection that contains either two classes that are a or are a subclass of two different `internal base classes <https://docs.python.org/3/library/stdtypes.html>`_ shall evaluate to ``Never``.
```

Comment:
This was moved from specification -> historical_specification as a means to help people view what's changed from when we started to now side by side in tools with rst rendering support since there has been too much discussion for people to catch up on in a reasonable amount of reading time. I have no intent on trying to determine prior intent for what was there before and maintain it. The currently relevant draft is still at specification.rst, apologies for any confusion that means of going about it may have caused.

I've just gone ahead and removed the file entirely, people using github's web interface can make that comparison by opening up different points in history, people comparing locally can use git worktree.